### PR TITLE
@eessex: addresses mobile cta banner qa items

### PR DESCRIPTION
**Pull request by xtina-starr**
This addresses the QA items in https://artsyproduct.atlassian.net/browse/GROW-844.

- Increases the CTA font-size
- Changes banner height
- Addresses animation delay
- Fixes z-index problem
- Passes the `redirect-to` as a query param
- Prevents multiple CTA banners from showing up on infinite scroll articles

### CODE REVIEW

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit c18b59a48bb661afdea894eea791fecda31305d2

https://deploy-preview-1136--artsy-reaction.netlify.com

**Comment by eessex** [comment]
Danger error isn't clear, but I'm noticing this needs a release commit.

File: src/Components/MinimalCtaBanner.tsx [hunk 1]
```diff
@@ -43,7 +43,9 @@ export class MinimalCtaBanner extends React.Component<
         >
           <Banner textColor={this.props.textColor}>
             <a href={this.props.href}>
-              <p>{this.props.copy}</p>
+              <Sans size="3" weight="medium" my={0.5}>
+                <p>{this.props.copy}</p>
```
Comment (eessex) [inline src/Components/MinimalCtaBanner.tsx]: 💯

**Comment by eessex** [comment]
This looks good to me when green!

**Pull request merged**